build.gradle

```diff
@@ -48,6 +50,7 @@ sourceSets {
         resources {
             srcDir 'src/main/resources'
             srcDir 'src/main/generated/resources'
+            exclude "assets/structurize/lang/default.json"
```

Comment:
shouldn't we exclude the manual file too?

Reply:
I save it to the build version of the manual one, so excluding it would leave us with no lang file at all.